frontend/src/components/dashboard/contactsHomePage/ContactHomePage.js

```diff
@@ -142,6 +145,12 @@ export class ContactHomePage extends Component {
     );
   };
 
+  handleContactTypeChange = () => {
+    this.props.change(FORM.CONTACT_ADVANCED_SEARCH, `party_name`, null);
```

Comment:
These don't need to be in backticks. Normal strings will do

Reply:
done